src/aspire/basis/steerable.py

```diff
@@ -132,10 +142,77 @@ def calculate_bispectrum(
 
         return B
 
-    def rotate(self, complex_coef, radians, refl=None):
+    def rotate(self, coef, radians, refl=None):
+        """
+        Returns coefs rotated by `radians`.
+
+        :param coef: Basis coefs.
+        :param radians: Rotation in radians.
+        :param refl: Optional reflect image (bool)
+        :return: rotated coefs.
+        """
+        # Covert radians to a broadcastable shape
+        if isinstance(radians, np.ndarray):
+            if len(radians) != len(coef):
```

Comment:
This would only work if `coef` has shape `(n, self.count,)` but wouldn't make sense for `(self.count,)` Reshape `coef` first?

Reply:
to `(-1, self.count)` i think

Reply:
Right, that is a case I didn't run into yet. Nice catch.

I'll atleast_2d(coef) (so if we start making ndim image/coef....)